src/BlazorWebView/src/Maui/Android/AndroidWebKitWebViewManager.cs

```diff
@@ -16,7 +17,7 @@ namespace Microsoft.AspNetCore.Components.WebView.Maui
 	/// An implementation of <see cref="WebViewManager"/> that uses the Android WebKit WebView browser control
 	/// to render web content.
 	/// </summary>
-	[SupportedOSPlatform("android23.0")]
+	[SupportedOSPlatform("android21.0")]
```

Comment:
We can remove these attributes now since we have a min version in .NET itself that is 21. .NET 6 does not run on lower than 21.

Reply:
Agreed